tendrl-commons.spec

```diff
@@ -25,6 +25,7 @@ Requires: python-urllib3
 Requires: python-six
 Requires: python-docutils
 Requires: python-yaml
+Requires: pytz
```

Comment:
namespaces rpm available in our fedora copr repo and the pytz is available in centos-base repo